### DISCUSSION TITLE
Link to local markdown files.

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#7] [General]
is it possible open and preview local Markdown file links in quicklook window? Vscode markdown preview can do this.

```
# Table of Contents

- [link to another markdown file](./another_markdown.md)
- [link to another markdown file 2](./another_markdown2.md)
```

**Comment by sbarex** [comment]
I'm sorry but it's not possible.
Quicklook preview do not allow interaction. After viewing the file it is not possible to invoke any action to the extension code.

**Reply by setanarut** [reply]
are you sure? Quicklook allow browsing remote pages.

`[Link to local markdown files.](https://github.com/sbarex/QLMarkdown/discussions/7)`


https://user-images.githubusercontent.com/36481442/103161315-ebe04a00-47f0-11eb-9359-b8993caeb762.mov

**Reply by relikd** [reply]
@hazarek I don't know how this works on your machine. For me, links in a QuickLook preview open in the default browser, not the preview itself. For a couple plugins indeed. Is there a special configuration flag to set?

**Reply by sbarex** [reply]
What would be the most appropriate behavior?
At the moment the links are opened within the preview.

The links to a markdown file, however, will not be formatted (because the extension is no longer operational while the webkit that manages the navigation between the links is working)

**Reply by setanarut** [reply]
> @hazarek I don't know how this works on your machine. For me, links in a QuickLook preview open in the default browser, not the preview itself. For a couple plugins indeed. Is there a special configuration flag to set?

No, i am using latest big sur 11.1

**Reply by sbarex** [reply]
Also I am using big sur

**Comment by sbarex** [comment]
In the release 1.0b5 I have added an option to choose if open link inside quicklook preview or in the default browser.

Unfortunately on big sur it doesn't seem to work due to some security limitations (can anyone help me?). Maybe in catalina it should go.

**Reply by relikd** [reply]
do I have to manually reload the plugin? Or is clicking on save enough? If latter, then no, it does not open the external link inside the preview window.

**Reply by relikd** [reply]
Ahh, ok, it just took a minute to update. Now the link is opening inside. Thanks!
